src/test/software/amazon/event/ruler/ACMachineTest.java

```diff
@@ -2564,4 +2757,16 @@ public void testAdditionalNameStateReuseSecondSubRuleSubsetOfFirstSubRule() thro
         assertEquals(1, matches.size());
         assertTrue(matches.contains("rule1"));
     }
+
+    private static Set<String> set(String ... strings) {
```

Comment:
nit: accidental space between String and ...

Reply:
Oh really? This was intentional. It's the way I've always done it. Is it generally considered better form to omit the space?

Reply:
don't think one is better than the other, but I thought we were using no space in the pkg. You comment made me check and realize I had it the wrong way. we do use space everywhere https://github.com/search?q=repo%3Aaws%2Fevent-ruler+...+path%3A%2F%5Esrc%5C%2F%2F&type=code, so ignore my previous comment.